### PR TITLE
COMMERCE 4586 6

### PR DESCRIPTION
**Pull request by ivicac**

### CODE REVIEW

**Comment by ivicac** [comment]
ci:test:sf

**Comment by ivicac** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-asset-categories-web/src/main/java/com/liferay/commerce/product/asset/categories/web/internal/display/context/CategoryCPDisplayLayoutDisplayContext.java [hunk 1]
```diff
@@ -87,6 +96,46 @@ public String getAddCategoryDisplayPageURL() throws Exception {
...
+				AssetVocabularyServiceUtil.getGroupVocabularies(
+					themeDisplay.getCompanyGroupId());
+			portletURL.setParameter(
+				"eventName", renderResponse.getNamespace() + "selectCategory");
+			portletURL.setParameter("singleSelect", "true");
+			portletURL.setParameter(
+				"vocabularyIds",
+				ListUtil.toString(
+					vocabularies, AssetVocabulary.VOCABULARY_ID_ACCESSOR));
+			portletURL.setWindowState(LiferayWindowState.POP_UP);
```
Comment (dsitu) [inline modules/apps/commerce/commerce-product-asset-categories-web/src/main/java/com/liferay/commerce/product/asset/categories/web/internal/display/context/CategoryCPDisplayLayoutDisplayContext.java:129]: @ivicac new line here.
```suggestion
			List<AssetVocabulary> vocabularies =
				AssetVocabularyServiceUtil.getGroupVocabularies(
					themeDisplay.getCompanyGroupId());

			portletURL.setParameter(
				"eventName", renderResponse.getNamespace() + "selectCategory");
			portletURL.setParameter("singleSelect", "true");
			portletURL.setParameter(
				"vocabularyIds",
				ListUtil.toString(
					vocabularies, AssetVocabulary.VOCABULARY_ID_ACCESSOR));
			portletURL.setWindowState(LiferayWindowState.POP_UP);
```

File: modules/apps/commerce/commerce-product-asset-categories-web/src/main/java/com/liferay/commerce/product/asset/categories/web/internal/portlet/action/EditCPDisplayLayoutMVCActionCommand.java [hunk 2]
```diff
@@ -123,21 +123,29 @@ protected void updateCPDisplayLayout(ActionRequest actionRequest)
 		long cpDisplayLayoutId = ParamUtil.getLong(
 			actionRequest, "cpDisplayLayoutId");
 
+		long classPK = ParamUtil.getLong(actionRequest, "classPK");
+
 		List<Long> classPKs = new ArrayList<>();
```
Comment (dsitu) [inline modules/apps/commerce/commerce-product-asset-categories-web/src/main/java/com/liferay/commerce/product/asset/categories/web/internal/portlet/action/EditCPDisplayLayoutMVCActionCommand.java:128]: Sort like this
```suggestion
		List<Long> classPKs = new ArrayList<>();

		long classPK = ParamUtil.getLong(actionRequest, "classPK");
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by ivicac** [comment]
@dsitu done

**Comment by ivicac** [comment]
ci:test:sf

**Review by dsitu — APPROVED** [review]

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**